### PR TITLE
cassandra-stress - Allow more configuration in cassandra-stress mk task

### PR DESCRIPTION
**Pull request by cscetbon**
Allow users to change several parameters in cassandra-stress run:
- cluster name (CLUSTER_NAME)
- datacenter (DC)
- username (USERNAME)
- password (PASSWORD)
- consistency level (CONSISTENCY_LEVEL)
- replication factor (REPLICATION_FACTOR)

See https://github.com/Orange-OpenSource/cassandra-k8s-operator/blob/39111610632c809d4b2492cd71094618926e552f/Makefile#L444-L447 for default values

As an example, the following command can be used :
`make cassandra-stress small DC=dc1 CLUSTER_NAME=cluster1`

### CODE REVIEW

File: tests/cassandra-stress/cassandra-stress-normal.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
       capabilities:
         add: ["IPC_LOCK"]
     command: ["/bin/sh"]
-    args: ["-c", "cassandra-stress 'user profile=/opt/cassandra-stress/normal_stress.yaml ops(insert=10,query_by_sub_id=8) duration=120m cl=one -node cassandra-demo-dc1.cassandra-demo -mode native cql3 user=bench password=monbench -rate threads=30 -pop seq=0..100M -tokenrange -graph file=/tmp/stress-normal.html' && echo END && while true ; do sleep 60; done"]
+    args: ["-c", "cassandra-stress 'user profile=/opt/cassandra-stress/normal_stress.yaml ops(insert=10,query_by_sub_id=8) duration=120m cl=one -node cassandra-demo -mode native cql3 user=bench password=monbench -rate threads=30 -pop seq=0..100M -tokenrange -graph file=/tmp/stress-normal.html' && echo END && while true ; do sleep 60; done"]
```
Comment (allamand) [inline tests/cassandra-stress/cassandra-stress-normal.yaml]: don't you want to configure also the node name ? here cassandra-demo, and also login/password used to connect ?

Comment (cscetbon) [reply tests/cassandra-stress/cassandra-stress-normal.yaml]: node name is already available https://github.com/Orange-OpenSource/cassandra-k8s-operator/pull/93/files#diff-b67911656ef5d18c4ae36cb6741b7965R462

for login/password, I think it's a good idea. I'll add that feature

Comment (cscetbon) [reply tests/cassandra-stress/cassandra-stress-normal.yaml]: Just added 2 new options `USERNAME` and `PASSWORD`.

**Comment by allamand** [comment]
@cscetbon there is a weird error in tests : `cmd/manager/main.go:194:11: undefined: metrics.ExposeMetricsPort`

**Comment by cscetbon** [comment]
Yes but why now ? That PR does touch that part ...

**Pull request merged**